hummingbot/connector/exchange/ascend_ex/ascend_ex_exchange.py

```diff
@@ -1046,14 +1071,22 @@ async def _process_order_message(self, order_msg: AscendExOrder):
 
             order_update = OrderUpdate(
                 exchange_order_id=order_msg.orderId,
-                trading_pair=await OBDS.trading_pair_associated_to_exchange_symbol(order_msg.symbol),
+                trading_pair=await AscendExAPIOrderBookDataSource.trading_pair_associated_to_exchange_symbol(order_msg.symbol),
```

Comment:
We should be passing the api_factory and the throttler instance in all places where we use the method to translate symbols to trading pairs and trading pairs to symbols